client/src/components/CreateGame/Lobby/Steps/Steps.tsx

```diff
@@ -7,6 +7,7 @@ import Rounds from "../Rounds/Rounds"
 import StartGame from "../StartGame/StartGame"
 import "react-toastify/dist/ReactToastify.css"
 import { ToastContainer, toast } from "react-toastify"
+import { host } from "../../../../utils/APIRoutes"
```

Comment:
Does React not provide a way to use relative path aliases, similar to Sveltekit's `$lib/utils/APIRoutes`?

Reply:
Probably? But I was just sticking with the convention that was being used in this project, since that's not really the focus of this PR :p